.github/workflows/wheels.yml

```diff
@@ -0,0 +1,30 @@
+name: Build wheels
+
+on: [push, pull_request]
```

Comment:
We might want to add

    push:
        tags:
            - '*'
Then it runs those jobs after a release for us to deploy the files on pypi.

Reply:
I could not make it work. Please have a go yourself - my toml/yaml foo is not strong enough.